packages/rocketchat-apps/client/admin/apps.html

```diff
@@ -14,80 +14,69 @@
-								<td>
-									<div class="rc-table-wrapper">
+							<div class="rc-apps-app manage" data-name="{{latest.name}}">
+								<div class="rc-apps-app-content">
+									<div class="rc-apps-app-icon">
 										{{#if latest.iconFileData}}
 											<div class="rc-table-avatar" style="background-image:url(data:image/png;base64,{{latest.iconFileData}})"></div>
 										{{else}}
 											<div class="rc-table-avatar" style="background-image:url({{latest.iconFileContent}})"></div>
 										{{/if}}
-										<div class="rc-table-info">
-											<span class="rc-table-title">
-												{{latest.name}}
-											</span>
-											{{#if latest.author.name}}
-												<span class="rc-table-subtitle">by {{latest.author.name}}</span>
-											{{/if}}
-										</div>
 									</div>
-								</td>
-								<td>{{latest.categories}}</td>
-								<td>
-									<p class="rc-table-title">
-										{{#if latest.summary}}
-											{{latest.summary}}
-										{{else}}
-											{{latest.description}}
+									<div class="rc-apps-app-info">
+										<div class="rc-apps-app-info-title">
+											{{latest.name}}
+										</div>
+										{{#if latest.author.name}}
+											<div class="rc-apps-app-info-subtitle">by {{latest.author.name}}</div>
 										{{/if}}
-									</p>
-									{{#if latest.summary}}
-										<p>
-											{{latest.description}}
-										</p>
-									{{/if}}
-								</td>
-								<td>
 
-									{{#if $eq latest._installed true}}
-										{{> icon icon="checkmark-circled" block="rc-icon--default-size"}}
-									{{/if}}
-									{{#if renderDownloadButton latest}}
-										<div class="rc-apps-marketplace__wrap-actions">
-											{{> icon block="rc-icon--default-size rc-icon" icon="loading"}}
-											<button class="js-install apps-installer" data-app="{{appId}}">
-												{{> icon block="installer rc-icon--default-size" icon="circled-arrow-down"}}
+										<!-- <p class="rc-table-title">
+											{{#if latest.summary}}
+												{{latest.summary}}
+											{{else}}
+												{{latest.description}}
+											{{/if}}
+										</p>
+										{{#if latest.summary}}
+											<p>
+												{{latest.description}}
+											</p>
+										{{/if}} -->
+									</div>
+									<div class="rc-apps-app-action">
+										{{#if $eq latest._installed true}}
+											<button class="js-configure rc-button rc-button--secondary rc-button--small" data-app="{{appId}}">
+												{{_ "Open"}}
 											</button>
-										</div>
-									{{/if}}
-								</td>
-							</tr>
+										{{/if}}
+										{{#if renderDownloadButton latest}}
+											<div class="rc-apps-app-action-install">
+												{{> icon block="rc-icon--default-size rc-icon" icon="loading"}}
+												<button class="js-install rc-button rc-button--primary rc-button--small" data-app="{{appId}}">
+													{{_ "Free"}}
```

Comment:
What about if there is a price?  How will that render?

Reply:
@geekgonecrazy will render the price `$2.00`, but this change is not predicting any API changes, if you already know the field I can add it here.

Reply:
It should be app.price with 0.00 on free ones.  

@graywolf336 has changes coming to move the api calls server side.  So not sure if effects the properties there any.  @graywolf336 can you confirm?